apps/src/weblab/WebLabView.jsx

```diff
@@ -152,6 +163,24 @@ class WebLabView extends React.Component {
               style={iframeStyles}
             />
             {!this.props.isProjectLevel && <CompletionButton />}
+            <BaseDialog
+              isOpen={this.props.shouldShowError}
+              handleClose={this.closeErrorDialog}
```

Comment:
can/should we use the `useUpdatedStyles` prop here? it'll make the dialog look more modern, but i also don't know if it'd be out of place here

Reply:
👍